src/lints/struct_with_no_pub_fields_changed.ron

```diff
@@ -0,0 +1,76 @@
+SemverQuery(
+    id: "struct_with_no_pub_fields_changed",
```

Comment:
Let's follow the naming convention. All other "struct changed to enum or union" lints end in "changed_type" so let's do that here as well. Just "changed" is vague and doesn't communicate what happened.

Remember to update the file name, the ID here, and the test crate.

Reply:
Idk why, I had in my head that these changes were just `_changed` from the initial looking through other lints.